test/over_react_test/custom_matchers_test.dart

```diff
@@ -510,9 +521,11 @@ main() {
             shouldPass(logs, hasNoLogs);
           });
 
-          test('when there are prop type errors', () {
-            shouldFail(logs, hasNoLogs, contains("has logs with value ['random log', 'nonsense', 'random log 2']"));
-          });
+          if (runtimeSupportsPropTypeWarnings()) {
+            test('when there are prop type errors', () {
+              shouldFail(logs, hasNoLogs, contains("has logs with value ['random log', 'nonsense', 'random log 2']"));
+            });
+          }
```

Comment:
Does this test need to change? It may be able to be more clear by decoupling it from `propTypes`, but since it's just checking against a list literal and the underlying `ddcOnly` value is false, I'd still expect it to be a valid test

Reply:
Addressed in https://github.com/Workiva/over_react_test/pull/104/commits/5e736cd83aa9091f0e0dd699c9e3d8020ce7fd4f